### ISSUE TITLE
`CACHE` doesn't lex correctly v0.6.6

### ISSUE THREAD

**Issue by jazzdan** [earthly/earthly#1632]
## Repro steps
1. Have earthly version v0.6.6:

```
earthly --version
earthly version v0.6.6 darwin/arm64; macOS
```

2. Clone earthly

```
git@github.com:earthly/earthly.git
```

3. Try to build the JS `CACHE` example:

```
$ cd examples/cache-command/npm/

$ earthly +docker

 1. Init 🚀
————————————————————————————————————————————————————————————————————————————————

           buildkitd | Found buildkit daemon as docker container (earthly-buildkitd)


 2. Build 🔧
————————————————————————————————————————————————————————————————————————————————

Error: build target: build main: bkClient.Build: failed to solve: resolve build context for target +docker: lexer error: Earthfile
syntax error: line 7:4 token recognition error at: 'CACHE '
syntax error: line 7:10 token recognition error at: '.'
syntax error: line 7:11 token recognition error at: '/'
syntax error: line 7:12 token recognition error at: 'node_'
syntax error: line 7:17 token recognition error at: 'modules
'
```

My earthly config file, FWIW:

```yaml
global:
    cache_size_mb: 40000
    local_registry_host: tcp://127.0.0.1:8371
    conversion_parallelism: 5
```

**Comment by vladaionescu** [comment]
We should probably add that example to our integration test suite.

**Comment by brandonSc** [comment]
I think that example may just be missing the feature toggle, i.e. `VERSION --use-cache-command 0.6`.

**Comment by jazzdan** [comment]
@brandonSc I tried that too :(

```
$ cat Earthfile
VERSION --use-cache-command 0.6

FROM node:17-alpine3.14
WORKDIR /app

deps:
    CACHE ./node_modules
    COPY package*.json .
    RUN npm install

code:
    FROM +deps
    COPY index.js .

docker:
    FROM +code
    ENTRYPOINT ["npm", "start"]
    SAVE IMAGE cache-example-npm:latest
    
 $ earthly +docker

 1. Init 🚀
————————————————————————————————————————————————————————————————————————————————

           buildkitd | Found buildkit daemon as docker container (earthly-buildkitd)


 2. Build 🔧
————————————————————————————————————————————————————————————————————————————————

Error: build target: build main: bkClient.Build: failed to solve: resolve build context for target +docker: lexer error: Earthfile
syntax error: line 7:4 token recognition error at: 'CACHE '
syntax error: line 7:10 token recognition error at: '.'
syntax error: line 7:11 token recognition error at: '/'
syntax error: line 7:12 token recognition error at: 'node_'
syntax error: line 7:17 token recognition error at: 'modules
'

**Comment by brandonSc** [comment]
Thanks @jazzdan , that's a bit more surprising as I'd expect it to work on v0.6.6. I'll dig into this some more.

**Comment by brandonSc** [comment]
An update on this issue:

It seems to be related specifically to the Homebrew version of Earthly. I'm able to build the cache examples using the Linux version, or by compiling the Mac OS M1 version manually. 

In fact, downloading and running the [earthly-darwin-arm64](https://github.com/earthly/earthly/releases/download/v0.6.6/earthly-darwin-arm64) binary directly from [the Github release page](https://github.com/earthly/earthly/releases/tag/v0.6.6) also seems to build the cache examples just fine. (@jazzdan this may work for you as well)

I'm concerned now there's a bigger problem with the Homebrew releases of Earthly. Although other features/fixes that were released in v0.6.6 seem to be present in that Homebrew release. Just the cache command that's not there for some reason (as far as I can tell so far anyway)

**Comment by vladaionescu** [comment]
This has been fixed in https://github.com/earthly/earthly/pull/1639 BTW. It'll be out in the next release.
